_data/tools.yml

```diff
@@ -635,6 +636,16 @@
     A design-first API managment platform with WYSIWYG API Designer for OpenAPI and AsyncAPI specifications.
   v2: true
   v3: true
+  
+- name: Frogment Editor
+  category: gui-editors
+  language: Desktop / SaaS
+  link: https://www.frogment.com
+  description: |
+    A free openAPI spec editor and linter that breaks down your spec into fragments to make editing easier and more intuituve.
+  v2: false
+  v3: true
+  v3_1: true
 
```

Comment:
Please can you add an empty line; it will prevent merge conflicts downstream PRs

Reply:
added

Reply:
Hi.  Can you review the changes